app/views/settings/ext_cert/index.html.slim

```diff
@@ -0,0 +1,135 @@
+= render 'settings/apply'
+
+h2 External Certificates
+
+p Enable the use of External SSL Certificates for CaaSP and Kubernetes services.  Certificates and matching keys must be in armored PEM format.
+.alert.alert-warning role='alert'
+ p Remember to add a Root CA Certificate to the System Wide Certificates corresponding to any new External Certificates
+
+= form_for :external_certificate, url: settings_ext_cert_index_path, method: :post do |f|
+  br
+  .panel
+    .panel-default
```

Comment:
When extracting to partials, do not forget to revert this to `.panel.panel-default`.